setup.py

```diff
@@ -18,12 +18,12 @@ def get_version(*file_paths):
                    version string
     """
     filename = os.path.join(os.path.dirname(__file__), *file_paths)
-    version_file = open(filename).read()
-    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
-                              version_file, re.M)
+    with open(filename, encoding="utf-8") as version_file_handle:
```

Comment:
Python 3 by default uses the "utf-8" encoding right? More or a curious question, why change from simple open() to specifying the encoding?

Reply:
I took a look, and it's not actually the default:

> In text mode, if encoding is not specified the encoding used is platform-dependent: [locale.getencoding()](https://docs.python.org/3/library/locale.html#locale.getencoding) is called to get the current locale encoding.

source: https://docs.python.org/3/library/functions.html#open

Also, pylint was complaining about the encoding being missing - that's the main reason I changed it there. :)

I did also find [PEP 686](https://peps.python.org/pep-0686/) which has been accepted, which will make utf-8 default from python 3.15 though.